pkg/util/http/tls.go

```diff
@@ -31,3 +31,18 @@ func ConfigureTls(httpClient *http.Client, serverCert string, clientCert string,
 	}
 	return nil
 }
+
+func ConfigureTlsWithoutServerVerification(httpClient *http.Client, clientCert string, clientKey string) error {
+	cert, err := tls.LoadX509KeyPair(clientCert, clientKey)
+	if err != nil {
+		return errors.Wrap(err, "could not create key pair from client cert and client key")
+	}
+
+	httpClient.Transport = &http.Transport{
+		TLSClientConfig: &tls.Config{
+			InsecureSkipVerify: true,
```

Comment:
Let's add a comment why we're doing this.

E.g., `since we're not going to pass any secrets to the server, we can skip validating its identity`